test/API/titerate.c

```diff
@@ -671,102 +676,100 @@ test_iter_group_large(hid_t fapl)
     /* Allocate & initialize array */
     names = (iter_info *)HDcalloc(sizeof(iter_info), (ITER_NGROUPS + 2));
     CHECK_PTR(names, "HDcalloc");
-#else
     (void)fapl;
-#endif
     /* Output message about test being performed */
     MESSAGE(5, ("Testing Large Group Iteration Functionality - SKIPPED for now due to no iteration restart "
                 "support\n"));
-#ifndef NO_ITERATION_RESTART
-    /* Create file */
-    file = H5Fcreate(DATAFILE, H5F_ACC_TRUNC, H5P_DEFAULT, fapl);
-    CHECK(file, FAIL, "H5Fcreate");
-
-    /* Create dataspace for datasets */
-    sid = H5Screate_simple(SPACE1_RANK, dims, NULL);
-    CHECK(sid, FAIL, "H5Screate_simple");
-
-    /* Create a bunch of groups */
-    for (i = 0; i < ITER_NGROUPS; i++) {
-        HDsnprintf(gname, sizeof(gname), "Group_%d", i);
+    if (vol_cap_flags_g & H5VL_CAP_FLAG_ITERATE) {
```

Comment:
This test should also check for at least H5VL_CAP_FLAG_FILE_BASIC, H5VL_CAP_FLAG_GROUP_BASIC, H5VL_CAP_FLAG_DATASET_BASIC and H5VL_CAP_FLAG_STORED_DATATYPES and we can removed the SKIPPED part of the message above.

Reply:
Okay.